reconcile.go

```diff
@@ -19,6 +19,10 @@ func (a *Autopilot) reconcile() error {
 	state := a.state
 	a.stateLock.Unlock()
 
+	if state == nil || state.Leader == "" {
```

Comment:
This is the protection for the reconciliation go routine against attempting to operate on invalid state.